### PR TITLE
Update `requirements-unlocked.txt`

### PR DESCRIPTION
**Pull request by joeflack4**
Adds missing packages.

This only affects local development,.

### CODE REVIEW

File: requirements-unlocked.txt [hunk 1]
```diff
@@ -1 +1,7 @@
+beautifulsoup4
```
Comment (twhetzel) [inline requirements-unlocked.txt:1]: @joeflack4 why is this needed?

**Review by twhetzel — APPROVED** [review]
Question inline, but approving.

File: requirements-unlocked.txt (hunk 1 above)
Comment (joeflack4) [reply requirements-unlocked.txt:1]: Very observant! I was wondering if you would notice this.

There is a utility which we haven't used in years called the code scraper. It is described in the "Additional tools" section of the readme. It allows us to see which MIMs had changes by a given month, and it pulls this information from the website.
